### DISCUSSION TITLE
How to use safelist like windicss?

### DISCUSSION THREAD

**Discussion by robin-dongbin** [unocss/unocss#763] [Q&A] [answered]
Hey guys, do you know how to use safelist like windicss?

**Comment by zyyv** [comment, marked as answer]
It also has a `safelist` config option. (`safelist: string[]`)


[playground](https://unocss.antfu.me/?html=DwEwlgbgBALgpgDxgWgMZwHbwE4D4BQUUA9hsiHALYCGGIUA7VGBgM4y0yNQBmArgBsBATyio%2B7YpTAAvagCMBcfMAD04CLiA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdOwb%2BBqzC6KhiwGY6%2BqoAkLRgFgCM9nSJAEz2CRYAzKnZACxcqr6cpACUQA&options=N4IgzgLgTglgxhEAuaBXApgXyA)

**Reply by robin-dongbin** [reply]
Thanks, it's not documented. I didn't think it would do that
